CHANGELOG.md

```diff
@@ -1,7 +1,21 @@
 # Changelog
 
 ## [0.12.6](https://github.com/brave/browser-laptop/releases/v0.12.6dev)
-- (TODO)
+- Added the ability to set Brave as your default browser ([#4937](https://github.com/brave/browser-laptop/issues/4937))([#4958](https://github.com/brave/browser-laptop/issues/4958))([#4935](https://github.com/brave/browser-laptop/issues/4935))([#4934](https://github.com/brave/browser-laptop/issues/4934))([#2105](https://github.com/brave/browser-laptop/issues/2105))([#4939](https://github.com/brave/browser-laptop/issues/4939))
+- Added the option to opt-in to reporting usage stats ([#4691](https://github.com/brave/browser-laptop/issues/4691))
+- Fixed various security issues ([#4747](https://github.com/brave/browser-laptop/issues/4747))([#4885](https://github.com/brave/browser-laptop/issues/4885))([#4913](https://github.com/brave/browser-laptop/issues/4913))([#4883](https://github.com/brave/browser-laptop/issues/4883))
+- Fixed error on invalid homepage URL ([#4974](https://github.com/brave/browser-laptop/issues/4974))
+- Fixed Autocomplete to behave more like Chrome / Firefox ([#3049](https://github.com/brave/browser-laptop/issues/3049))([#3235](https://github.com/brave/browser-laptop/issues/3235))([#4861](https://github.com/brave/browser-laptop/issues/4861))
```

Comment:
Please avoid direct comparisons to other browsers, and just say Fixed autocomplete handling so it works better.